advisor/src/main/kotlin/advisors/VulnerableCode.kt

```diff
@@ -144,7 +145,12 @@ class VulnerableCode(
             vulnerabilityUrls.map { extractVulnerabilityId(it) }
                 .map { async { service.getVulnerability(it) } }
                 .awaitAll()
-                .map { Vulnerability(it.cveId.orEmpty(), it.cvss ?: SEVERITY_UNDEFINED, URI(it.url)) }
-                .associateBy { "${it.url}" }
+                .map {
+                    Vulnerability(
+                        it.cveId.orEmpty(),
+                        listOf(VulnerabilityReference(URI(it.url), null, (it.cvss ?: SEVERITY_UNDEFINED).toString()))
```

Comment:
I believe what we really should do here instead of

    (it.cvss ?: SEVERITY_UNDEFINED).toString())

is

    it.cvss?.toString() ?: UNKNOWN_SEVERITY

and introduce `const UNKNOWN_SEVERITY = "Unknown"` to `VulnerabilityReference`. But that can be done later when touching this code anyway for the bulk-fetch implementation.